### ISSUE TITLE
Error: Tried to load an unregistered type.

### ISSUE THREAD

**Issue by kaduppg** [craftcms/cms#9711]
### Description

I've been having "Tried to load an unregistered type" problem every time I create a new channel section to use as an entry and then pull the data using GraphQL on my Next js project with Apollo Client or even on postman and GraphQL Playground.

### Steps to reproduce

1. Go to settings -> Sections -> New Section 
2. Fill up the name (eg. Test ) .... section type : Channel
3. Save and edit entry types
4. Default entry... On fields layout You can leave only with title for example and then Save it.
5. On Dashboard Go to Entries and create a new entry for Test section.
6. Go to GraphiQL on the dashboard
7. Try to query the new entry like so : 

query MyQuery {
  entries(section: "Test") {
   ... on test_default_Entry{
    title
  }
  }
}

On GraphiQL it is possible to see that the new entry  has been created.

8. Go to Postman or GraphQL Playground or any GraphQL Third party Client and try to run the same command. I got the following error : 

{
  "errors": [
    {
      "message": "Tried to load an unregistered type \"test_default_Entry\". This can indicate both a typo in the query or an issue with the schema used."
    }
  ]
}

So basically everything works pretty well on GraphiQL but not on Third Parties which is bad because I've been use Next Js to pull the data from craft cms using Apollo Client and I always end up having the same problem with new sections. 

PS: All the sections and entries I have created before using the same step by step worked pretty well, but for some reason now is not working anymore, might be a silly mistake but right now I have no idea why this is happening.

### Additional info

- Craft version: Craft Pro 3.7.8
- PHP version: 7.4.19
- Database driver & version:  MariaDB-1:10.6.3

**Comment by andris-sevcenko** [comment]
Do you also add the new entry type to the GraphQL schema that you're using in the third-party APIs?

Craft's GraphiQL IDE defaults to `Full schema`, which allows everything, however is not available outside of Craft API.

**Comment by kaduppg** [comment]
@andris-sevcenko ... That's why I mentioned before  "it might be a silly mistake" ... I completely forgot to add the new entry to the public Schema, sorry about that, and thanks a lot for the answer \o/.
